src/atom_finder/classifier/operator-precedence.clj

```diff
@@ -78,7 +78,9 @@
       (order-insensitive-opt-combination (sort combination))
 
       ;;SPECIAL CASE 1: Comma operator in any combination;
-      (some (partial = :comma) combination)))
+      (some (partial = :comma) combination)
+      ;SPECIAL CASE 2: note that the pair name is not [:bitwise_bin :bitwise-bin]
```

Comment:
I'm glad you put a comment here, but what does it mean. What's the difference between bitwise_bin, bitwise-bin, and bitwise--bin?